src-ui/lib/components/BooksTable.svelte

```diff
@@ -1,28 +1,36 @@
 <script lang="ts">
-  import type { Book, BookWithoutId } from '$lib/types/book.js'
+  import type { Book, NewBook } from '$lib/types/book.js'
   import 'bulma/css/bulma.css'
   import onScan from 'onscan.js'
   import type { Action } from 'svelte/action'
-  import { getByISBN } from '../../lib/openLibrary.js'
-  import { books } from '../state/Books.svelte'
+  import { getByISBN } from '$lib/openLibrary.js'
+  import { createBooksStore } from '$lib/state/Books.svelte'
   import BooksTableRow from './BooksTableRow.svelte'
 
-  const initialBook: BookWithoutId = {
-    isbn10: '1234567890',
-    title: 'Hunting Prince Dracula',
-    tags: ['Young Adult', 'Fiction'],
-    authors: ['Kerri Maniscalco'],
-    hasRead: true,
-  }
+  let booksStorePromise = createBooksStore()
 
-  if (books.value.length == 0) {
-    books.add(initialBook)
-  }
+  $effect(() => {
+    const initialBook: NewBook = {
+      isbn10: '1234567890',
+      title: 'Hunting Prince Dracula',
+      tags: ['Young Adult', 'Fiction'],
+      authors: ['Kerri Maniscalco'],
+      hasRead: true,
+    }
+    void booksStorePromise.then(async (booksStore) => {
+      if (booksStore.value.length === 0) {
+        await booksStore.add(initialBook)
+      }
+    })
+  })
```

Comment:
We can probably remove this in favor of making the simulate scan button more fully featured / having a large random list of IDs. But keeping it in for now.